app/models/profile.py

```diff
@@ -873,7 +873,7 @@ def parse_graph(self) -> Graph:
                 premis_representation_sparql,
                 "w",
             ) as f:
-                f.write(profile_template.render(path=path))
+                f.write(profile_template.render(path=path, rep_folder=path.parts[-4]))
```

Comment:
Die `path.parts[-4]` gebruik ik een paar in de code. Maar dit is best gevoelig. Het idee was om de METS te parsen en gebruik te maken van de informatie over de file/folder structuur. Dan kan de `glob` code ook weg.